src/backend/marsha/core/tests/models/test_migration_livesession.py

```diff
@@ -14,7 +14,7 @@
 )
 
 
-class LiveSessionModelsMigrationgLiveAttendanceTestCase(TestCase):
+class LiveSessionModelsMigratingLiveAttendanceTestCase(TestCase):
```

Comment:
I think we should keep `Migration`. There is a typo and should be fix by removing the extra `g`

Reply:
Actually I should have asked, I hesitated between `Migration` and `Migrating` as the initial one was `Migrationg`, my bad, I gonna fix it up.

Reply:
FYI, migrationg is formal marseillais